tests/unit/QuantType.spec.js

```diff
@@ -216,6 +216,31 @@ describe('QuantType.vue', () => {
 
   })
 
+  describe('library10XVDJ', () => {
+
+    beforeEach(() => {
+      cmp = Vue.extend(QuantType)
+      quantType = new cmp({propsData: { quantType: 'library10XVDJ'}})
+    })
+
+    it('must have the correct options', () => {
+      expect(quantType.$data).toEqual(quantTypes['library10XVDJ'])
+    })
+
+    it('has the correct well type', () => {
+      expect(quantType.$data.wellType).toEqual('TS10XVDJ')
```

Comment:
If wellType is an data attribute on the quantType component, I think `quantType.wellType` should work!?